### ISSUE TITLE
Editing Interface for editing frames

### ISSUE THREAD

**Issue by joeyklee** [shiffman/BodyBlock#22]
As a first pass for a simple editing interface for image frames. Sorry for the eye-sore of a wireframe, but working quickly to jot down some basic thoughts!

![MANUAL-EDITING](https://user-images.githubusercontent.com/3622055/83977763-0bee0180-a8d1-11ea-80b4-01bb4b89f2cb.png)

**Comment by shiffman** [comment]
Something that could be useful (especially if a particular model drops a frame or two) is an option to "copy overlay from previous (or subsequent?) frame. I suppose instead of actually drawing on the video like a "stamp", we could consider all of the overlay elements to be manipulable elements like you can grow / shrink of adjust position for any given frame.

**Comment by joeyklee** [comment]
@shiffman - Our approach at the moment is definitely a much simpler approach to just writing directly to the images. We could instead create a data model for storing the coordinates that would be used to generate canvas elements that could be manipulated. This would be mellow for the face bounding boxes, but for the bodyPix segmentation results ,this might be much harder. Or is there a good way to do this?

**Comment by shiffman** [comment]
Yes, I agree. I think this is probably a much later step and better served by learning what things people actually need to do with the videos. I agree that this would work much more easily for faces which can just be `x,y,w,h` objects. 

Another simpler idea would be to save every frame as two images/canvases, the video and a transparent overly. This way a transparent overlay could be duplicated from one frame to another to save manual repetitious editing (for example if a face that stays in one place isn't being caught for many frames.)
